src/Extensions/Hosting/Certificates/CertificateReader.cs

```diff
@@ -79,7 +79,8 @@ public IEnumerable<X509Certificate2> GetCertificatesByCommonName(string commonNa
 
 		private CertificateInformation[] LoadCertificates(StoreName storeName, bool refreshCache)
 		{
-			if (refreshCache || !m_certificatesCache.TryGetValue(storeName, out CertificateInformation[]? certificates))
+			CertificateInformation[]? certificates;
```

Comment:
What was the reason behind this change?

Reply:
I was getting error "Converting null literal or possible null value to non-nullable type" in IDE and made the auto-fix from the list of actions. On second inspection (now) I believe the correct fix would be to switch back to in line out variable declaration but remove the '?' from the certificates type, as it does not match the signature of TryGetValue method.
Would you like me to apply it? Or rollback the fix altogether?

Reply:
Strange it does not show any warning to me, what IDE are you using? It's probably something to do with type marking as nullable, logically if the item is absent then it would return null. 
I would suggest removing this change since it's unrelated to the issue

Reply:
I am using JetBrains Rider.
I will remove the change

Reply:
Okay, that explains the difference. 
I've checked the option of marking the type as non-nullable and as expected it causes a warning in VS.

So it's probably better to apply your change since that would work in both VS and Rider without warnings to avoid future problems for contributors using any of those.